appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/deployment/node/runtime/EjbNode.java

```diff
@@ -390,7 +393,7 @@ public Node writeDescriptor(Node parent, String nodeName, EjbDescriptor ejbDescr
      * @param parent node for the information
      * @param descriptor the descriptor containing the generated info
      */
-    private void writeGenClasses(Node parent, EjbDescriptor ejbDescriptor) {
+    private static void writeGenClasses(Node parent, EjbDescriptor ejbDescriptor) {
```

Comment:
Why did `writeGenClasses` instance method change to a static method?

Reply:
Because it can be a `static` method - something we can reason about as a function. 

This helps a reader to see quickly (and with IDE also on call site) that this is side effect free (except from doing static side-effect which hopefully should be rare). It also is a hint that whatever is going on in this method is independent of the class defining it so it might either be a utility that could be in a general utility class or it should maybe be a member of one of its argument classes instead).